types/k6/experimental/browser.d.ts

```diff
@@ -2078,7 +2078,7 @@ export interface Mouse {
      * @param y The y position.
      * @param options The click options.
```

Comment:
Do we also need to add `@returns` here? Some other methods (non-mouse/keyboard) have it.

Reply:
I personally don't find `@returns` helpful if they just communicate the same thing as the return type.

One example I've seen was a `@returns` comment saying "An array of Cookies`, which isn't really helpful. I'd save such comments for when you need to explain some which behavior would cause which value. Something like "If no page has been navigated to, this method will be null. Otherwise it will return the current href" (this example is not based on any real method, just something I made up).

In this case, I feel the return type communicates that it will resolve once the operation has finished. 

Or put another way: the return type itself is documentation.

Reply:
Thanks for the info 🙇 Let's keep this in mind no longer use `@returns` 👍